CHANGELOG.md

```diff
@@ -3,6 +3,135 @@ All notable changes to this project will be documented in this file.
+ - doc: all examples in the documentation are now generated from running the code, so they are current. ([#7457])
+
+
+### Changed
+
+ - protocol: We now send current peers our changed gossip (even if they set timestamp_filter otherwise), not just on reconnect. ([#7554])
+ - protocol: we now always ask the first peer for all its gossip. ([#7512])
+ - connectd: I/O optimizations to significantly speed up larger nodes. ([#7365])
+ - connectd: now should use far less CPU on large nodes. ([#7365])
+ - plugins: the `fetchinvoice` plugin has been combined into the `offers` plugin. ([#7456])
+ - close: We no longer attempt to publish a unilateral close that'd fail anyway when we witness a close onchain. ([#7447])
+ - pay: Improved logging should facilitate debugging considerably. ([#7418])
+ - reckless: option flags are now position independent. ([#7484])
+ - plugins: now allows date and time sqlite functions. ([#7467])
+ - splice: added outnum of new funding output to splice_signed RPC command ([#7465])
+ - lightningd: we wait for bitcoind if it has somehow gone backwards (as long as header height is still ok). ([#7342])
+ - wallet: The channel status is printed when loading it from the DB ([#7354])
+ - JSON-RPC: `listclosedchannels`, `listpeerchannels`, `openchannel_update`, `openchannel_init`, `fundchannel`, `fundchannel_start` and `multifundchannel`: `channel_type` array `names` now contains "anchors" instead of "anchors_zero_fee_htlc_tx". ([#7388])
+ - JSON-RPC: Do not return the contents of invalid parameters in error messages, refer to logs (use 'check' to get full error messages) ([#7420])
+ - lightningd: `--list-features-only` now lists "option_anchors" instead of "option_anchors_zero_fee_htlc_tx". ([#7388])
+ - updated Bitcoin to v27.1 and Elements to v23.2.1 ([#7436])
+ - update libwally to 1.3.0 ([#7480])
+
+
+### Deprecated
+
+Note: You should always set `allow-deprecated-apis=false` to test for changes.
+
+ - JSON-RPC: `listpeers` `features` array string "option_anchors_zero_fee_htlc_tx": use "option_anchors" (spec renamed it). ([#7388])
+ - config: the --experimental-onion-messages option is ignored (on by default). ([#7455])
+ - config: the --experimental-anchors option is ignored (on by default since v24.02). ([#7454])
+ - pyln-client: `category`, `description` and `long_description` for RPC commands are deprecated now. ([#7520])
+
+
+### Removed
+
+ - JSON-RPC: `sendonionmessage` (was experimental only, use `injectonionmessage`) ([#7461])
+ - JSON-RPC: `autocleaninvoice` command (deprecated v22.11, EOL v24.02) ([#7298])
+ - plugins: `estimatefees` returning feerates by name (e.g. "opening"); deprecated in v23.05. ([#7380])
+
+
+### Fixed
+
+ - protocol: we can now open unannounced channels with LND nodes again. ([#7564])
+ - plugins: pay can now pay a bolt12 invoice even if we, ourselves, are the head of the blinded path within it. ([#7461])
+ - renepay: finalized routes have to be processed and ([#7357])
```

Comment:
and.. .?

Reply:
also, renepay is EXPERIMENTAL

Reply:
Also the message was truncated. May be change to this
```suggestion
 - renepay: bugfix, un-reserve routes that have been completed or failed ([#7357])
```

Reply:
Changed the message as suggested.